packages/ibm-products/src/components/Datagrid/Datagrid/DatagridToolbar.tsx

```diff
@@ -181,30 +201,30 @@ const DatagridBatchActionsToolbar = (datagridState, width, ref) => {
   );
 };
 
-const DatagridToolbar = ({ ariaToolbarLabel, ...datagridState }) => {
+const DatagridToolbar = ({
+  ariaToolbarLabel,
+  ...datagridState
+}: DatagridToolbarProps & DataGridState) => {
   const ref = useRef(null);
   const { width } = useResizeObserver(ref);
   const { DatagridActions, DatagridBatchActions, batchActions, rowSize } =
     datagridState;
-
   const getRowHeight = rowSize || 'lg';
-
   return batchActions && DatagridActions ? (
     <div
       ref={ref}
       className={cx([blockClass, `${blockClass}--${getRowHeight}`])}
     >
       <TableToolbar aria-label={ariaToolbarLabel}>
-        {DatagridActions && <DatagridActions {...datagridState} />}
-        {DatagridBatchActionsToolbar &&
-          DatagridBatchActionsToolbar(datagridState, width, ref)}
+        {<DatagridActions {...datagridState} />}
```

Comment:
Maybe this was removed when debugging the storybook loading issue, but this line removed `DatagridActions &&`.
```suggestion
        {DatagridActions && <DatagridActions {...datagridState} />}
```